docs/source/algorithms/PolarizationCorrectionFredrikze-v1.rst

```diff
@@ -41,7 +57,67 @@ Entry in group  Measurement
 4               :math:`I_{aa}`
 ==============  ================
 
+Output from Polarized Neutron Reflectivity
+------------------------------------------
+
+The output of this algorithm is a :ref:`WorkspaceGroup <WorkspaceGroup>`, the resulting :ref:`WorkspaceGroup <WorkspaceGroup>` will have 2 entries, corresponding to the output specified spin state orders, if no specific output spin states are specified, it will be formatted as follows:
+
+==============  ================
+Entry in group  Measurement
+==============  ================
+1               :math:`I_{p}`
+2               :math:`I_{a}`
+==============  ================
+
+Usage
+-----
+
+.. include:: ../usagedata-note.txt
+
+**Example - PolarizationCorrectionFredrikze (PNR)**
+
+.. testcode:: PolarizationCorrectionFredrikze (PNR)
+
+    # import mantid algorithms, numpy and matplotlib
+    from mantid.simpleapi import *
+    import matplotlib.pyplot as plt
+    import numpy as np
+
+    run = Load("POLREF00004699.nxs")
+    eff = Load("fredrikze_efficiencies_37125.nxs")
+    run = ConvertUnits(run, "Wavelength")
+    eff = ConvertUnits(eff, "Wavelength")
+    eff = RebinToWorkspace(eff, run.getItem(0), True)
+    out = PolarizationCorrectionFredrikze(run, eff, "PNR", InputSpinStates="pp, pa, ap, aa", OutputSpinStates="p, a")
+
```

Comment:
There should probable be a `.. testoutput::` here as in the one from the wildes docs. That way we can make sure this usage example consistently performs correctly.